doc/api/test.md

```diff
@@ -1579,9 +1579,10 @@ added:
 Enables timer mocking for the specified timers.
 
 * `timers` {Array} An optional array containing the timers to mock.
-  The currently supported timer values are `'setInterval'` and `'setTimeout'`.
-  If no array is provided, all timers (`'setInterval'`, `'clearInterval'`, `'setTimeout'`,
-  and `'clearTimeout'`) will be mocked by default.
+  The currently supported timer values are `'setInterval'`, `'setTimeout'`
```

Comment:
These are now all timers... do we still need/want to list them here?

Reply:
it's still missing process.nextTick and schedulers. Should we?

Reply:
Oh forgot about it. maybe we should only highliht the unavailable timer then?

Reply:
Again, interval, timeout, and immediate are all timers, and using the term without mentioning them all will cause confusion.